Plugins/UnityClientPlugin/MediaEngineUWP/Shared/MediaEnginePlayer.h

```diff
@@ -179,11 +185,14 @@ ref class MEPlayer: public MediaEngineNotifyCallback
     void StartTimer();
     void StopTimer();	
     void OnTimer();
+	void OnVSyncTimer();
```

Comment:
nit: these don't align with the other signatures (at least on github) - is there a weird spacing issue here?

Reply:
github issue, looks fine in VS.